src/PIL/_typing.py

```diff
@@ -29,4 +32,4 @@ def read(self, __length: int = ...) -> _T_co:
 StrOrBytesPath = Union[str, bytes, "os.PathLike[str]", "os.PathLike[bytes]"]
 
 
-__all__ = ["FileDescriptor", "TypeGuard", "StrOrBytesPath", "SupportsRead"]
```

Comment:
Looks like something might have gone wrong in the merge commit here; I don't think these new symbols should have been removed from `__all__`?

Reply:
Oh, my bad. I've force-pushed a fixed merge commit.